server/sonar-server/src/main/java/org/sonar/server/issue/index/IssueIndexer.java

```diff
@@ -63,12 +63,17 @@ public AuthorizationScope getAuthorizationScope() {
   }
 
   @Override
-  protected long doIndex(long lastUpdatedAt) {
-    return doIndex(createBulkIndexer(false), lastUpdatedAt, null);
+  public Set<IndexTypeId> getIndexTypes() {
+    return ImmutableSet.of(INDEX_TYPE_ISSUE);
+  }
+
+  @Override
+  public void initializeOnStartup() {
+    doIndex(createBulkIndexer(true), (String) null);
   }
 
   public void indexAll() {
```

Comment:
This method should be dropped. It is used only in medium tests, which should call `initializeOnStartup()` (not a pb if "large" mode is enabled IMO).